templates/pgadmin/meta.yaml

```diff
@@ -0,0 +1,88 @@
+name: Pgadmin
```

Comment:
Use the proper casing for the names: pgAdmin

Make sure you check their website or repo to see how the names should be cased.